bftengine/tests/simpleTest/scripts/simpleTest.py

```diff
@@ -457,6 +458,15 @@ def main():
         if n is None or r is None or f is None or c is None or cl is None:
             print("Unable to parse BFT parameters")
             return -1
+        if 3 * f + 2 * c + 1 != n:
+            g_logger.error("N = 3f + 2c + 1 is not satisfied")
+            return -1
+        max_r = max(4, multiprocessing.cpu_count() * 4)
+        if r > max_r:
+            g_logger.error("max number of running replicas on your machine is "
```

Comment:
Is the limit of 2 running replicas per core just to keep this test from being too slow, or is this limit required for correctness?